CHANGELOG.md

```diff
@@ -2,6 +2,32 @@
 
 # Changelog
 
+## v1.11.7 (2019-12-13)
+    Allow ADD to use http src
+    Bump to c/storage v.1.15.3
+    install.md: update golang dependency
+    imgtype: reset storage opts if driver overridden
+    Start using containers/common
+    overlay.bats typo: fuse-overlays should be fuse-overlayfs
+    chroot: Unmount with MNT_DETACH instead of UnmountMountpoints()
+    bind: don't complain about missing mountpoints
+    imgtype: check earlier for expected manifest type
+    Vendor containers/storage fix
+    Vendor containers/storage v1.15.1
+    Add history names support
```

Comment:
This would qualify for a minor bump to `v1.12.0`.

Reply:
I agree, bump the second number,  If we only fix bugs then minor.  If new features like history names then the middle.  If changes in API then major.

Reply:
Missed that.  Will kill this then.  @rhatdan should I wait for the /var/tmp to land before a new kit and does the history change cause us to NOT want to get this in Podman 1.7.0?

Reply:
I don't think you should wait for this.